Modules/itertoolsmodule.c

```diff
@@ -4585,16 +4585,16 @@ cycle(p) --> p0, p1, ... plast, p0, p1, ...\n\
 repeat(elem [,n]) --> elem, elem, elem, ... endlessly or up to n times\n\
 \n\
 Iterators terminating on the shortest input sequence:\n\
-accumulate(p[, func]) --> p0, p0+p1, p0+p1+p2\n\
+accumulate(p, func=None) --> p0, p0+p1, p0+p1+p2\n\
```

Comment:
I would like to leave this as-is.   Accepting "None" is an implementation detail that wasn't an intended part of the API, isn't documented, and isn't tested.